docs/ExpectAPI.md

```diff
@@ -646,6 +647,45 @@ test('the house has my desired features', () => {
 });
 ```
 
+### `toHaveProperty(propPath, value)`
```

Comment:
Meh, I can't come up with something descriptive enough. I recall Ember's `get()` uses the same notation and they call it [`keyName`](https://github.com/emberjs/ember.js/blob/v2.11.0/packages/ember-metal/lib/property_get.js#L50)